### ISSUE TITLE
Term.replaced_by raises KeyError

### ISSUE THREAD

**Issue by FrozenTux** [althonos/pronto#44]
Hello,

I'm using pronto to parse the Human Phenotype Ontology (HPO) which is in obo format ([file here](http://purl.obolibrary.org/obo/hp.obo)). When I'm given an obsolete term, I would like to automatically fall back to the term that replaced it via the replaced_by field.

However, whenever I try to access the replaced_by property of a Term, I get a strange KeyError.

Example of an obsolete term in HPO:
```
[Term]
id: HP:0000284
name: obsolete Abnormality of the ocular region
is_obsolete: true
replaced_by: HP:0000315
```

Example of code used with the error:
```
>>> from pronto import Ontology
>>> ontology = Ontology("http://purl.obolibrary.org/obo/hp.obo")
__main__:1: UnicodeWarning: unsound encoding, assuming ISO-8859-1 (73% confidence)
>>> term = ontology["HP:0000284"]
>>> term
Term('HP:0000284', name='obsolete Abnormality of the ocular region')
>>> term.obsolete
True
>>> term.replaced_by
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "XXX/lib/python3.7/site-packages/pronto/term.py", line 402, in replaced_by
    return frozenset({ontology.get_term(t) for t in termdata.replaced_by})
  File "XXX/lib/python3.7/site-packages/pronto/term.py", line 402, in <setcomp>
    return frozenset({ontology.get_term(t) for t in termdata.replaced_by})
  File "XXX/lib/python3.7/site-packages/pronto/utils/meta.py", line 81, in newfunc
    return func(*args, **kwargs)
  File "XXX/lib/python3.7/site-packages/pronto/ontology.py", line 373, in get_term
    raise KeyError(id)
KeyError: 'H'
```
The target term exists in the ontology and is correctly loaded:
```
>>> ontology["HP:0000315"]
Term('HP:0000315', name='Abnormality of the orbital region')
```

Is it a problem with the ontology file or with pronto ? And can I just get the raw value of the replaced_by field and handle it myself ?

Thank you in advance

Versions used : Python 3.7.5, pronto 1.1.2

**Comment by althonos** [comment]
Hi @FrozenTux,

this is an issue with `pronto`, in particular the `OboParser`. It was fixed in f761a6b920436b9c6ca323976dd8b0e41dfa3277 but I still haven't released the patch. I'll try to do it shortly!

**Comment by althonos** [comment]
(Hopefully) fixed in `v1.1.3`.

**Comment by FrozenTux** [comment]
I have confirmed that it is now working. Thanks a lot !
